### DISCUSSION TITLE
Community of describo?

### DISCUSSION THREAD

**Discussion by hakor** [describo/describo.github.io#13] [General]
Dear developers

I just found the describo tool and started to evaluating it as a potential candidate for a RDM System. 
RO-Crate and therefore describo are often discussed in the RDM community. 

But I was really surprised that up to know only one committer (thanks for that excellent work, marco!!) commited code to this repo. 

Is this correct? 

Harald

**Comment by marcolarosa** [comment]
Hi Harald,

Yes - I'm the primary developer and I maintain the component, web and desktop versions (as well as this site). Recently, I've had a lot of help from @beepsoft with the component. Specifically, @beepsoft and team have created the web component / react component version of the crate builder as well as adding internationalisation.

I use the describo component in a number of my projects and maintain the web and desktop versions as a contribution to the research community in my own time as an open source developer.

If there are features you would like added then please describe them in the relevant issue tracker. And if you are in a position to contribute code then please do so.

Thanks!
Marco
